deployment/arm/region.json

```diff
@@ -1,307 +1,475 @@
-                "format": "xml"
-            }
-        },
-        {
-            "type": "Microsoft.ApiManagement/service/diagnostics/loggers",
-            "apiVersion": "2018-01-01",
-            "name": "[concat(parameters('apimName'), '/applicationinsights/', parameters('appInsightsName'))]",
-            "dependsOn": [
-                "[resourceId('Microsoft.ApiManagement/service/diagnostics', parameters('apimName'), 'applicationinsights')]",
-                "[resourceId('Microsoft.ApiManagement/service', parameters('apimName'))]",
-                "[resourceId('microsoft.insights/components', parameters('appInsightsName'))]"
-            ],
-            "properties": {
-                "loggerType": "applicationInsights",
-                "credentials": {
-                    "instrumentationKey": "{{Logger-Credentials-5df9447791f0e91e700024ea}}"
-                },
-                "isBuffered": true,
-                "resourceId": "[resourceId('microsoft.insights/components', parameters('appInsightsName'))]"
-            }
-        },
-        {
-            "type": "Microsoft.Storage/storageAccounts",
-            "apiVersion": "2019-04-01",
-            "name": "[parameters('storageActName')]",
-            "location": "[parameters('location')]",
-            "sku": {
-                "name": "Standard_LRS",
-                "tier": "Standard"
-            },
-            "kind": "StorageV2"
-        }
-    ]
-}
+{
```

Comment:
noob q: is there a way to better track arm template diff?

Reply:
Yes - my machine reformatted the whitespace.... this hopefully won't happen again.